boards/posix/native_posix/native_posix.dts

```diff
@@ -65,10 +65,9 @@
 		};
 	};
 
-	uart0: uart@1 {
+	uart0: uart {
```

Comment:
Can we still support multiple uarts in this case?

Reply:
Could add a non-zero dummy size value is `reg`, e.g. `reg = <1 1>` (and then `reg = <2 1>`, etc.)

We're just trying to get rid of the explicit setting of the register size to 0, so that we can warn for it (might be a sign that something is off in other cases).

Reply:
ok thx for the info